src/main/java/com/zendesk/maxwell/MaxwellAbstractRowsEvent.java

```diff
@@ -275,21 +279,75 @@ public Object getData(String string) {
 		return list;
 	}
 
-	public List<JSONObject> toJSONObjects() {
-		ArrayList<JSONObject> list = new ArrayList<>();
+	static final JsonFactory factory = new JsonFactory();
```

Comment:
Maybe call this `jsonFactory` to distinguish it from all the other factories that might spring into being at any moment, this being Java?